compiler-rt/include/sanitizer/container_overflow_defs.h

```diff
@@ -0,0 +1,275 @@
+//===-- sanitizer/container_overflow_defs.h ---------------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// Public sanitizer interface defs for container overflow checks.
+//===----------------------------------------------------------------------===//
+
+#ifndef SANITIZER_CONTAINER_OVERFLOW_DEFS_H
+#define SANITIZER_CONTAINER_OVERFLOW_DEFS_H
+
+// Windows allows a user to set their default calling convention, but we always
+// use __cdecl
+#ifdef _WIN32
+#define SANITIZER_CDECL __cdecl
```

Comment:
```suggestion
#define _SANITIZER_CDECL __cdecl
```

This basically needs to be done throughout, for all identifiers. The preprocessor does not differentiate between different "kinds" of tokens.